packages/itinerary-body/src/defaults/access-leg-description.tsx

```diff
@@ -111,6 +113,23 @@ export default function AccessLegDescription({
             place: placeContent
           }}
         />
+      ) : isTransferLeg ? (
+        <FormattedMessage
+          defaultMessage="Transfer, wait {duration}"
+          description="Summarizes an transfer leg"
+          id="otpUi.AccessLegBody.transfer"
+          values={{
+            duration: intl.formatMessage(
+              {
+                id: "otpUi.ItineraryBody.common.durationShort"
+              },
+              {
+                approximatePrefix: false,
+                ...coreUtils.time.toHoursMinutesSeconds(leg.duration)
```

Comment:
Nit: I see a call to `formatDuration` above, could we use that here?

Reply:
Both behave slightly differently! Also `formatDuration` is not guaranteed to be present